Various improvements

treacheroust opened the pull request:
Add color to pieces (looks nice!)
Save solutions images, and keep searching for more solutions after finding one (run it all night!)
Detect duplicate solutions, and don't save. 

Requires Pillow python module to be installed.
Requires ghostscript to be installed on the machine and in path.
This is all related to the save_to_file code.  Tkinter allows the a canvas widget to be save in a postscript file.  Pillow provides code to convert from postscript to other file types, but it requires ghostscript to be installed, and be in the path.   (I already had it installed, but I needed to add it to the path via "System Properties" "Environment Variables..." on my windows machine.)

notroot commented:
Can you do a requirements file for the libraries that need to be pip installed?

treacheroust commented:
I added a requirements file.  Also, since the initial pull request comment, I added the following:
- each piece has a unique color
- the process keeps searching for more solutions (even after finding one) and saves them to files
- changed the algorithm to find ALL of the solutions (it determined there were 233)
- I committed the solutions to the repo for reference
